STYLEGUIDE.md

```diff
@@ -380,23 +380,23 @@ Use lodash and jQuery libraries in `/apps`.
 
 ## CSS
 
-Default: https://github.com/thoughtbot/guides/tree/master/style/sass
-
-Fallback: https://css-tricks.com/sass-style-guide
-
-Fallback: http://css-tricks.com/css-style-guides
+On the frontend, some lint and style rules are enforced by [Stylelint](https://stylelint.io/). You can check your changes locally by running `yarn lint:scss` from the `apps` directory. Some rules have been disabled for now, but we are moving towards enabling all of the rules that are part of the Stylelint standard configuration.
 
+Some key points:
+- Use SCSS modules over CSS-in-JS. The module file takes the name "my-component.module.scss" and lives in the same directory as the component.
+- Use kebab-case (not camelCase nor snake_case) for separating words in IDs, classes, mixins and filenames. 
```

Comment:
Haven't we been writing class names using camelCase so far?

Reply:
Huh. We should definitely norm on this 😭 .

The [standard](http://jasonbuckboyer.com/playground/use-hyphens-in-css/#supporting) is to use hyphens (kebab-case), but clearly we haven't been doing that with our modules. The default Stylelint settings (which are disabled here) are to use kebab case (see [selector-class-pattern](https://stylelint.io/user-guide/rules/list/selector-class-pattern) and  [selector-id-pattern](https://stylelint.io/user-guide/rules/list/selector-id-pattern).

So I guess the question becomes: How do we want to norm?

Reply:
As far as filenames go, we seem to have `.jsx` files alongside corresponding `.module.scss` files in the same directories, but the former use camel case while it sounds like the latter should use kebab case.  It feels like there could be a benefit to using camel case for both?  Otherwise it's harder to match the corresponding files in a directory listing or IDE.

Reply:
I agree there is benefit to using camelCase for both. I laid out what I see are some of the complexities of this question here https://github.com/code-dot-org/code-dot-org/pull/51202#discussion_r1161761769.

At this point, I'm all for camelCase, and I'm also in favor of using a lint rule to enforce this going forward. It'd be a helluva bunch of exceptions but not quite as many as enforcing kebab-case

Reply:
Dave also put together an idea (https://github.com/code-dot-org/code-dot-org/pull/51202#pullrequestreview-1376365001) that'd use kebab-case